File: Solution/WebSolution/Layered/TFLOBL/Entities/ProjectMgmt/ProjectMembershipDetails.cs
namespace Transflower.TFLPortal.TFLOBL.Entities;

public class ProjectMembershipDetails : ProjectMembership
{
    public int UserId { get; set; }
    public DateTime HireDate { get; set; }
    public int ReportingId { get; set; }
    public double Salary { get; set; }
    public int EmployeeId { get; set; }
    
}

File: Solution/UISolution/Draft/src/app/shared/draft/projects/Models/projectmembershipdetails.ts

export class ProjectMembershipDetails {
    constructor(
        public userId: number,
        public hireDate: number,
        public reportingId: string,
        public projectId: number,
        public employeeId: number,
        public projectRole: string,
        public projectAssignDate: string,
        public firstName: string,
        public lastName: string,

    ) { }
}